src/ui/public/filter_bar/filter_bar_click_handler.js

```
@@ -25,7 +25,7 @@ export function FilterBarClickHandlerProvider(Notifier, Private) {
         aggConfigResult = event.point.aggConfigResult;
       }
 
-      if (aggConfigResult) {
+      if (aggConfigResult && $state) {
```

Maybe fail earlier in this method, if `$state` is not defined.